configuration/connect-to-ms-exchange.md

```diff
@@ -0,0 +1,233 @@
+
+In the TemplateVM:
+
+`sudo tar -xvzf ~/QubesIncoming/<AppVM>/davmail.*.tgz -C /opt`
+`sudo mv /opt/davmail* /opt/davmail`
+
+tell your template VM where to find the GPG Vault VM 
+
+`export QUBES_GPG_DOMAIN=my-vault`
+
+test access via SplitGPG (this command should list your secret-keys in the Vault-AppVM)
+
+`qubes-gpg-client -K` 
+
+
+Shutdown TemplateVM and create a new AppVM based on the TemplateVM
+in dom0
+
+`qvm-shutdown --wait t-mail`
+`qvm-create --template t-mail --label blue my-workmail`
+
+Open Qubes Setting for the new AppVM and increase storage capacity
+Private storage max. size: 20480 MiB
+
+Start AppVM and continue configuration there
+
+`qvm-run --auto my-workmail gnome-terminal`
+
+Start davmail
+
+`/opt/davmail/davmail.sh &`
+
+- Main Tab: Echange Protocol: Auto
+- Main Tab: OWA (Exchange) URL: https://owa.domain.com/owa/
+- Main Tab: [ ] Local POP port
+- Advanced Tab: Default windows domain: <YOUR-DOMAIN>
+- Advanced Tab: [x] Disable Update Check
+
+tell your AppVM VM where to find the GPG Vault VM 
+
+`echo "export QUBES_GPG_DOMAIN=my-vault" >> /home/user/.bashrc`
+
+allow TemplateVM to acces SplitGPG Vault-AppVM
+in dom0
+
+`echo "my-workmail my-vault allow" >> /etc/qubes-rpc/policy/qubes.Gpg`
+
+Test SpliGPG in the AppVM
+Close and restart Terminal, so that the variable QUBES_GPG_DOMAIN will be set
+Check if connection via SplitGPG works
+
+`qubes-gpg-client --list-secret-keys`
+
+
+## Setting up Thunderbird with Davmail
+Start Thunderbird in the AppVM
+
+`thunderbird &`
+
+- Create new account (use existing mail)
+- choose manual config
+- IMAP localhost 1143 Autodetect Autodetect
+- SMTP localhost 1025 Autodetect Autodetect
+- Username: <YOUR-USERNAME>
+- Click on Re-Test and accept any certificate warning
+- (Checking the fingerprints!)
+- Click on Re-Test, then Done
+- Warning about unencrypted transfer from localhost
+- [x] I understand the risks
+
+
+## Setting up neomutt and offlineimap with Davmail
+See also: https://hobo.house/2015/09/09/take-control-of-your-email-with-mutt-offlineimap-notmuch/ assuming that you have installed and configured Davmail (see above)
+
+### Install packages in the TemplateVM
+install neomutt and offlineimap in the Template VM
+dom0:
+
+`qvm-run --auto t-mail gnome-terminal`
+
+in the Template VM:
+```
+sudo dnf install dnf-plugins-core
+sudo dnf copr enable flatcap/neomutt
+sudo dnf install neomutt dialog offlineimap git notmuch
+```
+additional tools for a good neomutt experience
+
+`sudo dnf -y install w3m qutebrowser mupdf`
+`shutdown -h now`
+
+restart AppVM if it was running before and you installed new packages in the Template VM
+dom0:
+
+`qvm-shutdown --wait --quiet my-workmail`
+`qvm-run --auto my-workmail gnome-terminal`
+
+Start & configure davmail (see above)
+
+`/opt/davmail/davmail.sh &`
+
+### Setting up neomutt
+Clone mutt-wizard
+
+`git clone https://github.com/LukeSmithxyz/mutt-wizard.git ~/.config/mutt`
```

Comment:
Again, some warning about source verification could be useful. Also, you may want to link to https://www.qubes-os.org/doc/mutt/ for those unwilling to download random script from the internet...

Reply:
As the idea was to provide a simple(r) approach getting (neo)mutt but also Exchange configured I think it might make sense to split this how-to into sections:

1) Qubes + Davmail -> to get Exchange connected
2) Qubes + Neomutt -> How to get neomutt working under Qubes OS using the neomutt-wizard script but also by creating an own configuration from scratch, covering three examples:
Googlemail / Outlook.com / Corporate Exchange (connected via DavMail, referencing to the 1st how-to).

There is no need describing everything in one How-to, what do you think?

Reply:
Sounds good.